packages/expo-application/src/Application.ts

```diff
@@ -39,20 +119,40 @@ export enum ApplicationReleaseType {
   APP_STORE = 5,
 }
 
+// @docsMissing
 export async function getIosApplicationReleaseTypeAsync(): Promise<ApplicationReleaseType> {
   if (!ExpoApplication.getApplicationReleaseTypeAsync) {
     throw new UnavailabilityError('expo-application', 'getApplicationReleaseTypeAsync');
   }
   return await ExpoApplication.getApplicationReleaseTypeAsync();
 }
 
+// @docsMissing
 export async function getIosPushNotificationServiceEnvironmentAsync(): Promise<string> {
   if (!ExpoApplication.getPushNotificationServiceEnvironmentAsync) {
     throw new UnavailabilityError('expo-application', 'getPushNotificationServiceEnvironmentAsync');
   }
   return await ExpoApplication.getPushNotificationServiceEnvironmentAsync();
 }
 
+// @needsAudit
+/**
+ * Gets the time the app was installed onto the device, not counting subsequent updates. If the app
+ * is uninstalled and reinstalled, this method returns the time the app was reinstalled.
+ * - On iOS, this method uses the [`NSFileCreationDate`](https://developer.apple.com/documentation/foundation/nsfilecreationdate?language=objc)
+ * of the app's document root directory.
+ * - On Android, this method uses [`PackageInfo.firstInstallTime`](https://developer.android.com/reference/android/content/pm/PackageInfo.html#firstInstallTime).
+ * - On web, this method returns `null`.
+ *
+ * @return Returns a `Promise` that fulfills with a `Date` object that specifies the time the app
+ * was installed on the device.
+ *
+ * # Example
+ * ```ts
+ * await Application.getInstallationTimeAsync();
+ * // 2019-07-18T18:08:26.121Z
+ * ```
```

Comment:
Examples like this are what I'd like to keep out of our code and instead write by hand in our docs, interleaved with the autogenerated API reference.

Reply:
The whole auto generation code and `APISection` component from the beginning was written to render as single block. There is no option to interleave it or insert into it hand written markdown code at this moment. 

Unfortunately, this was not a requirement earlier, and to be honest, at this moment of time I'm not even sure how to properly implement this. Maybe the markdown code can be passed as an custom formatted object (with keys matching the method/prop/enum/type name), but I have a feeling that it would complicate a bit the conversion process.

Reply:
let's get an initial version of this working and then circle back to how we can make it more flexible

Reply:
OK. If there are bigger code snippets, I lean towards dropping them instead of setting an example.